tests/test_plugin.py

```diff
@@ -38,3 +38,17 @@ def test_plugin_attach_sanic_app_extension(
         warnings.simplefilter("error")
         endpoint.run("actions")
     manager.hook.attach_sanic_app_extensions.assert_called_once_with(app=app_mock)
+
+
+def test_plugins_not_found(monkeypatch):
+    # Mock the import statement to raise ModuleNotFoundError
+    monkeypatch.setitem(
+        __builtins__, "import", MagicMock(side_effect=ModuleNotFoundError)
+    )
+
+    # Call the method under test
```

Comment:
We should actually try to call the plugin_manager, similar to this approach, then there's no need to monkeypatch (because the rasa_sdk_plugins is not installed in the CI):

```
with caplog.at_level(logging.DEBUG) as caplog:
        plugin_manager()

assert "No plugins found" in caplog.text
```